FastSimulation/Tracking/python/InitialStep_cff.py

```diff
@@ -0,0 +1,57 @@
+import FWCore.ParameterSet.Config as cms
+
+# import the full tracking equivalent of this file
+import RecoTracker.IterativeTracking.InitialStep_cff
+
+# trajectory seeds
+import FastSimulation.Tracking.TrajectorySeedProducer_cfi
+initialStepSeeds = FastSimulation.Tracking.TrajectorySeedProducer_cfi.trajectorySeedProducer.clone(
+    simTrackSelection = FastSimulation.Tracking.TrajectorySeedProducer_cfi.trajectorySeedProducer.simTrackSelection.clone(
+        pTMin = 0.4,
+        maxD0 = 1.0,
+        maxZ0 = 999,
+        ),
+    minLayersCrossed = 3,
+    nSigmaZ = RecoTracker.IterativeTracking.InitialStep_cff.initialStepSeeds.RegionFactoryPSet.RegionPSet.nSigmaZ,
+    originpTMin = RecoTracker.IterativeTracking.InitialStep_cff.initialStepSeeds.RegionFactoryPSet.RegionPSet.ptMin,
+    originRadius = RecoTracker.IterativeTracking.InitialStep_cff.initialStepSeeds.RegionFactoryPSet.RegionPSet.originRadius,
+    layerList = RecoTracker.IterativeTracking.InitialStep_cff.initialStepSeedLayers.layerList.value()
+    )
+
+# track candidates
+import FastSimulation.Tracking.TrackCandidateProducer_cfi
+initialStepTrackCandidates = FastSimulation.Tracking.TrackCandidateProducer_cfi.trackCandidateProducer.clone(
+    SeedProducer = cms.InputTag("initialStepSeeds"),
+    MinNumberOfCrossedLayers = 3
+    )
+
+# tracks
+initialStepTracks = RecoTracker.IterativeTracking.InitialStep_cff.initialStepTracks.clone(
+    Fitter = 'KFFittingSmootherWithOutlierRejection',
+    TTRHBuilder = 'WithoutRefit',
+    Propagator = 'PropagatorWithMaterial'
+    )
+
+# vertices
+firstStepPrimaryVertices = RecoTracker.IterativeTracking.InitialStep_cff.firstStepPrimaryVertices.clone()
+
+# simtrack id producer
+initialStepSimTrackIds = cms.EDProducer("SimTrackIdProducer",
```

Comment:
@shchauha 

define simtrackids after final selection
i believe you rather need the final track collection (see comment above)